I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00keymap.rpy", line 413, in script
    python hide:
  File "renpy/common/00keymap.rpy", line 413, in script
    python hide:
  File "renpy/common/00keymap.rpy", line 413, in <module>
    python hide:
  File "renpy/common/00keymap.rpy", line 432, in _execute_python_hide
    renpy.save("_reload-1", "reload save game")
AttributeError: 'NoneType' object has no attribute 'stores'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "F:\Ren'Py\renpy-7.2.1-sdk\renpy\bootstrap.py", line 313, in bootstrap
    renpy.main.main()
  File "F:\Ren'Py\renpy-7.2.1-sdk\renpy\main.py", line 426, in main
    if renpy.parser.report_parse_errors():
  File "F:\Ren'Py\renpy-7.2.1-sdk\renpy\parser.py", line 2817, in report_parse_errors
    renpy.display.error.report_parse_errors(full_text, error_fn)
  File "F:\Ren'Py\renpy-7.2.1-sdk\renpy\display\error.py", line 187, in report_parse_errors
    error_fn=error_fn,
  File "F:\Ren'Py\renpy-7.2.1-sdk\renpy\game.py", line 281, in invoke_in_new_context
    return callable(*args, **kwargs)
  File "F:\Ren'Py\renpy-7.2.1-sdk\renpy\display\error.py", line 45, in call_exception_screen
    return renpy.ui.interact(mouse="screen", type="screen", suppress_overlay=True, suppress_underlay=True)
  File "F:\Ren'Py\renpy-7.2.1-sdk\renpy\ui.py", line 289, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "F:\Ren'Py\renpy-7.2.1-sdk\renpy\display\core.py", line 2690, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "F:\Ren'Py\renpy-7.2.1-sdk\renpy\display\core.py", line 3146, in interact_core
    renpy.exports.reload_script()
  File "F:\Ren'Py\renpy-7.2.1-sdk\renpy\exports.py", line 1748, in reload_script
    renpy.game.call_in_new_context("_save_reload_game")
  File "F:\Ren'Py\renpy-7.2.1-sdk\renpy\game.py", line 332, in call_in_new_context
    return renpy.execution.run_context(False)
  File "F:\Ren'Py\renpy-7.2.1-sdk\renpy\execution.py", line 892, in run_context
    context.run()
  File "renpy/common/00keymap.rpy", line 413, in script
    python hide:
  File "renpy/common/00keymap.rpy", line 413, in script
    python hide:
  File "F:\Ren'Py\renpy-7.2.1-sdk\renpy\ast.py", line 896, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "F:\Ren'Py\renpy-7.2.1-sdk\renpy\python.py", line 1929, in py_exec_bytecode
    exec bytecode in globals, locals
  File "renpy/common/00keymap.rpy", line 413, in <module>
    python hide:
  File "renpy/common/00keymap.rpy", line 432, in _execute_python_hide
    renpy.save("_reload-1", "reload save game")
  File "F:\Ren'Py\renpy-7.2.1-sdk\renpy\loadsave.py", line 398, in save
    roots = renpy.game.log.freeze(None)
  File "F:\Ren'Py\renpy-7.2.1-sdk\renpy\python.py", line 1826, in freeze
    self.complete(False)
  File "F:\Ren'Py\renpy-7.2.1-sdk\renpy\python.py", line 1456, in complete
    self.current.stores[name], self.current.delta_ebc[name] = sd.get_changes(begin)
AttributeError: 'NoneType' object has no attribute 'stores'

Windows-8-6.2.9200
Ren'Py 7.2.1.457
 
Thu Apr 04 12:04:21 2019
